src/api/get.rs

```diff
@@ -145,9 +145,6 @@ pub struct GetGalleryImagesResponse {
 #[utoipa::path(
     tag = "images",
     context_path = "/api", 
-    params(
```

Comment:
Should we also remove the `query_params` expected at the function `get_user_images` if we remove this one?